.rubocop.yml

```diff
@@ -4,10 +4,17 @@
 # https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
 # https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
 
-Rails:
-  Enabled: true
```

Comment:
On a similar point to previous, were we relying on any Rails default cops?

Reply:
... continuing from my Rails comment above, there are 2 requires to _active_support_ (though I don't see any method calls). Then, adding Rails cop back would make sense, which could mean sharing the .rubocop.yml could make sense; but how?